static/app/views/starfish/views/spans/selectors/domainSelector.tsx

```diff
@@ -173,8 +173,8 @@ const LABEL_FOR_MODULE_NAME: {[key in ModuleName]: ReactNode} = {
   cache: t('Domain'),
   vital: t('Domain'),
   queue: t('Domain'),
-  screen: t('Domain'),
-  startup: t('Domain'),
+  screen_load: t('Domain'),
+  app_start: t('Domain'),
```

Comment:
```suggestion
  screenLoad: t('Domain'),
  appStart: t('Domain'),
```

Similar to my other comment, I'm not exactly sure what's being mapped here to use the `Domain` string

Reply:
These values are used as labels for some specific dropdowns in http and db modules (see "SQL Command" and "Table in screeshot). They aren't used for the remaining modules, but are required in this constant due to the `{[key in ModuleName]: ReactNode}` typing. I included default Action and Domain values for these remaining modules.

<img width="625" alt="image" src="https://github.com/getsentry/sentry/assets/58920989/4e49b75f-0694-4dca-b812-3b7bfc2f0555">

Reply:
I see, that's kind of annoying that you had to make this change. I say we can fix the typing here later so we don't have to update this list every time we add a new module but I'm fine with keeping it as-is for now

Reply:
Agreed, will see if I can fix that in the follow up when I move instrumentation to the panel itself